client.py

```diff
@@ -0,0 +1,20 @@
+import socket
+import sys
+
+from kvstore.constants import INPUT_PROMPT
+
+def main():
+    while True:
+        i = input(INPUT_PROMPT)
+
+        s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+        s.connect(('localhost', 65432))
```

Comment:
i think i should be using a unix socket instead of an internet socket, but not sure how to make it work with this configuration

Reply:
Should be exactly the same but with `socket.AF_UNIX` instead of `socket.AF_INET`. If you tried that and there was an issue let me know and we can go deeper.